src/js/media-time-display.js

```diff
@@ -125,7 +132,10 @@ class MediaTimeDisplay extends MediaTextDisplay {
     ) {
       const timesLabel = formatTimesLabel(this);
       updateAriaValueText(this);
-      this.container.innerHTML = timesLabel;
+      // Only update if it changed, timeupdate events are called a few times per second.
+      if (timesLabel !== this.#slot.innerHTML) {
+        this.#slot.innerHTML = timesLabel;
+      }
```

Comment:
suggestion(non-blocking):
```suggestion
      if (timesLabel !== this.#slot.textContent) {
        this.#slot.textContent = timesLabel;
      }
```